src/software/ai/hl/stp/tactic/get_behind_ball/get_behind_ball_fsm.h

```diff
@@ -95,10 +95,12 @@ struct GetBehindBallFSM
             // inside it when taking the chip.
             Point behind_ball_vertex_A = event.control_params.ball_location;
             Point behind_ball_vertex_B =
-                behind_ball_vertex_A + behind_ball.normalize(size_of_region_behind_ball) +
-                behind_ball.perpendicular().normalize(size_of_region_behind_ball / 8);
+                behind_ball_vertex_A +
+                behind_ball.normalize(size_of_region_behind_ball + 2) +
```

Comment:
where did this 2 come from?

is this change necessary for this proof of concept?